docker/Dockerfile

```diff
@@ -7,4 +7,15 @@
 # docker build -f docker/Dockerfile -t oncotree-container:oncotree-tag-name . 
 
 FROM openjdk:8-jdk-alpine
+
+# Install autossh
+RUN apk add --no-cache autossh
+
+# Add the SSH configuration file
+RUN mkdir -p /root/.ssh
+RUN chmod 700 /root/.ssh
+RUN echo "Host *" > /root/.ssh/config && \
+    echo "    StrictHostKeyChecking no" >> /root/.ssh/config && \
+    echo "    UserKnownHostsFile=/dev/null" >> /root/.ssh/config
```

Comment:
I think I remember that there is a configmap we have which provide the needed credentials in a mounted filesystem
I hope autossh is resilient enough to keep this connection open. I have no experience with this tool.